docs/source/user-guide/compatibility.md

```diff
@@ -40,19 +40,23 @@ Comet currently has three distinct implementations of the Parquet scan operator.
 | `native_datafusion`     | This implementation delegates to DataFusion's `ParquetExec`.                                                                                                                           |
 | `native_iceberg_compat` | This implementation also delegates to DataFusion's `ParquetExec` but uses a hybrid approach of JVM and native code. This scan is designed to be integrated with Iceberg in the future. |
 
-The new (and currently experimental) `native_datafusion` and `native_iceberg_compat` scans are being added to
```

Comment:
Removed the "experimental" status

Reply:
I backtracked and added this back because we still have test failures in the Spark SQL tests